src/api/spec/controllers/webui/project_controller_spec.rb

```diff
@@ -917,4 +917,64 @@
       it { expect(package.attribs.where(attrib_type: attribute_type)).to be_empty }
     end
   end
+
+  describe 'POST #new_release_request' do
+    let(:project) { create(:maintenance_incident_project) }
```

Comment:
Could you use 

``
  let(:apache_maintenance_incident_project) { create(:maintenance_incident_project, name: 'ApacheMI') }
``
?